mtags/src/main/scala/scala/meta/internal/mtags/MtagsIndexer.scala

```diff
@@ -142,7 +142,7 @@ trait MtagsIndexer {
       definition: m.Position,
       kind: s.SymbolInformation.Kind,
       properties: Int
-  ): Unit = {
+  ): String = {
```

Comment:
I changed `addSignature` to return the added symbol so we can use those symbols as a next owner.